pandas/core/indexes/datetimes.py

```diff
@@ -2231,7 +2231,7 @@ def date_range(start=None, end=None, periods=None, freq='D', tz=None,
 
     Notes
     -----
-    Of the three parameters: ``start``, ``end``, and ``periods``, exactly two
+    Of the three parameters: ``start``, ``end`` and ``periods``, exactly two
     must be specified.
 
     To learn more about the frequency strings, please see `this link
```

Comment:
I'd personally move these notes to the extended summary. I see the Notes more for implementation details, and the extended summary for the general information everybody can use.

Reply:
Alright , I am working on this.